src/compose/network.ts

```diff
@@ -200,8 +198,9 @@ export class Network {
 		// Check if every ipam config entry has both a subnet and a gateway
 		_.each(_.get(config, 'ipam.config', []), ({ subnet, gateway }) => {
 			if (!subnet || !gateway) {
-				throw new InvalidNetworkConfigurationError(
-					'Network IPAM config entries must have both a subnet and gateway',
+				log.warn(
+					'Network IPAM config entries must have both a subnet and gateway defined.' +
+						' Your app containers might not be able to reach your container through the defined network address otherwise.',
```

Comment:
let's discuss if we want to log a warning since this might have unexpected behaviour when the SV skips over creating this network and other parts of the application need it.